tests/rptest/tests/timequery_test.py

```diff
@@ -458,6 +458,66 @@ def query_slices(tid):
 
         assert not any([e > 0 for e in errors])
 
+    @cluster(num_nodes=4)
+    # @parametrize(cloud_storage=True, spillover=False)
+    # @parametrize(cloud_storage=True, spillover=True)
```

Comment:
nit: remove?

Reply:
I will re-enable this in the next PR https://github.com/redpanda-data/redpanda/pull/18097